### DISCUSSION TITLE
Mantine with NextJS 13

### DISCUSSION THREAD

**Discussion by Pham-Tuan-Phat** [mantinedev/mantine#2902] [Q&A] [answered]
Hi,

I am very impressed with Mantine and I am learning how to apply it in my NextJS applications (version 12).
However, NextJS 13 is in beta. I would like to know if Mantine will be compatible with NextJS 13 when it is officially released.
I have this question because NextJS 13's application structure is much different from previous versions. NextJS 13 also has a lot of interesting things to consider.

Thanks.

**Comment by rtivital** [comment, marked as answer]
You can follow this issue – https://github.com/mantinedev/mantine/issues/2815
